deeppavlov/core/commands/train.py

```diff
@@ -188,6 +190,9 @@ def train_evaluate_model_from_config(config: [str, Path, dict], to_train=True, t
             }
 
             print(json.dumps(report, ensure_ascii=False))
+            all_reports.append(report)
+
+    return all_reports
```

Comment:
This will return a `NameError` `if not (train_config['validate_best'] or train_config['test_best'])`

Reply:
fixed